api/serializers.py

```diff
@@ -51,3 +52,10 @@ class ContactSerializer(serializers.ModelSerializer):
     class Meta:
         model = Contact
         fields = ["fullname", "email", "brand_tag"]
+
+
+class BrandSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Brand
+        # returns all fields from model
```

Comment:
currently `bank` api fails if we try to update a bank with countries. can you just add `countries = MultipleCountryField(required=True)` before class meta